### PR TITLE
[SYSDM] Enhance Spanish translation

### PR DESCRIPTION
**Pull request by elhoir**
Some strings were not translated. Fix it.

### CODE REVIEW

File: dll/cpl/sysdm/lang/es-ES.rc [hunk 1]
```diff
@@ -348,13 +348,13 @@ BEGIN
...
-    IDS_USERPROFILE_MANDATORY "Mandatory"
-    IDS_USERPROFILE_CONFIRM_DELETE "Are you sure you want to delete %s's profile?"
-    IDS_USERPROFILE_CONFIRM_DELETE_TITLE "Confirm Delete"
-    IDS_USERPROFILE_ACCOUNT_UNKNOWN "Account Unknown"
-    IDS_USERPROFILE_ACCOUNT_DELETED "Account Deleted"
-    IDS_USERPROFILE_TYPE_TEXT "When %s logs onto this computer, should the operating system use the roaming profile or just the locally cached copy of the roaming profile."
+    IDS_USERPROFILE_ROAMING "Móvil"
+    IDS_USERPROFILE_MANDATORY "Obligatorio"
+    IDS_USERPROFILE_CONFIRM_DELETE "¿Está seguro de que quiere borrar el perfil %s'?"
+    IDS_USERPROFILE_CONFIRM_DELETE_TITLE "Confirme borrado"
```
Comment (julcar) [inline dll/cpl/sysdm/lang/es-ES.rc:354]: I would preffer "Confirmar eliminación"

Comment (julenuri) [reply dll/cpl/sysdm/lang/es-ES.rc:354]: So do I, or maybe other formula, like "confirmar borrado", because coherence of the same style.

**Review by julenuri — APPROVED** [review]

File: dll/cpl/sysdm/lang/es-ES.rc (hunk 1 above)
Comment (elhoir) [reply dll/cpl/sysdm/lang/es-ES.rc]: I think "borrado" is more emphatic to the end-user than "eliminación". He will probably pay more attention.

File: dll/cpl/sysdm/lang/es-ES.rc [hunk 2]
```diff
@@ -369,6 +369,6 @@ BEGIN
     IDS_HWPROFILE_ALREADY_IN_USE "El nombre de perfil ya está en uso."
     IDS_HWPROFILE_PROFILE "Perfil"
     IDS_HWPROFILE_WARNING "Advertencia"
-    IDS_ENVIRONMENT_WARNING "Any changes that have been done will be discarded and the variable's value will be edited as text."
-    IDS_ENVIRONMENT_WARNING_TITLE "System Properties"
+    IDS_ENVIRONMENT_WARNING "Cualquier cambio será descartado y el valor de la variable será editado como texto"
```
Comment (HBelusca) [inline dll/cpl/sysdm/lang/es-ES.rc:372]: Missing full-stop
```suggestion
    IDS_ENVIRONMENT_WARNING "Cualquier cambio será descartado y el valor de la variable será editado como texto."
```

**Review by julenuri — APPROVED** [review]

**Comment by Swyter** [comment]
@elhoir A bit late, but my comments from the other day weren't being shown because pending reviews turn out to be personal drafts.

**Pull request merged**